helm-charts/infisical-pki-issuer/values.yaml

```diff
@@ -0,0 +1,34 @@
+controllerManager:
+  manager:
+    args:
+    - --metrics-bind-address=:8443
+    - --leader-elect
+    - --health-probe-bind-address=:8081
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+    image:
+      repository: docker.io/infisical/pki-issuer
+      tag: latest
```

Comment:
nit: i think the common pattern is to release a new app version for helm version along with each release. we can do this as a later ticket